scripts/globals/spells/victory_march.lua

```diff
@@ -20,12 +20,12 @@ function onSpellCast(caster,target,spell)
         power = power + math.floor((sLvl+iLvl-300) / 7)
     end
 
-    if (power >= 96) then
-        power = 96
+    if (power >= 163) then
+        power = 163
     end
 
     local iBoost = caster:getMod(dsp.mod.MARCH_EFFECT) + caster:getMod(dsp.mod.ALL_SONGS_EFFECT)
-    power = power + iBoost*16
+    power = power + Boost*16
```

Comment:
Should that not have remained iBoost? Otherwise you're changing it a different variable than the one two lines above that contains the song modifiers.